projectlicense.md

```diff
@@ -0,0 +1,81 @@
+=======
+License
+=======
+
+pandas is distributed under a 3-clause ("Simplified" or "New") BSD license.
+
+pandas license
+==============
+
+Copyright (c) 2012-2016, pandas Development Team
+All rights reserved.
```

Comment:
This ends up as one line, reading `...pandas Development Team All rights reserved.` Add a period after "Team"? 

Same for the next two copyrights...